src/Services/PatientService.php

```diff
@@ -39,15 +39,6 @@ public function __construct()
         parent::__construct('patient_data');
         $this->patientValidator = new PatientValidator();
     }
-    public function setPid($pid)
```

Comment:
removed these in favors of requiring a `$pid` parameter for `getOne` and `getAll`